R/read_delim.R

```diff
@@ -72,6 +72,7 @@ NULL
 #'   supplied any commented lines are ignored _after_ skipping.
 #' @param n_max Maximum number of lines to read.
 #' @param guess_max Maximum number of lines to use for guessing column types.
+#'   Will never use more than the number of lines read.
```

Comment:
```suggestion
#'   Will never use more than `n_max`, the number of lines read.
```

Reply:
Yeah, I don't think we need to be that precise.